pkgs/development/python-modules/cadquery/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyparsing
+, opencascade_oce
+, pythonocc-core
```

Comment:
@costrouc, you've probably already noticed this, but CadQuery use [their own branch of pythonocc-core](https://github.com/CadQuery/pythonocc-core) and the current one from nixpkgs won't work.